student.rb

```diff
@@ -0,0 +1,17 @@
+require './person'
+
+# Student class that extends from person
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permision: true)
+    super(age, name, parent_permision: parent_permision)
+    @classroom = classroom
+  end
+
+  def play_hooky
+    '¯\\(ツ)/¯'
+  end
+
+  def add_classroom(classroom)
+    classroom.add_student(self)
+  end
```

Comment:
- [ ] Remember that the `Student` class belongs to the `Classroom` class. Therefore, this `add_classroom` method should reflect that. Every classroom object created has a `students` array. To reflect this relationship, you need to push the classroom object which is the parameter for the method into the classroom's student's array unless the classroom is already in the array. You can read this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) to have a better understanding. 👍